docs/Tiered-Storage-on-Tachyon.md

```diff
@@ -1,90 +1,208 @@
+* `tachyon.worker.tieredstore.level0.dirs.quota=100GB` sets the quota for the ramdisk to be `100GB`.
+* `tachyon.worker.tieredstore.level1.alias=HDD` configures the second tier to be HDD.
+* `tachyon.worker.tieredstore.level1.dirs.path=/mnt/hdd1,/mnt/hdd2,/mnt/hdd3` configures 3 separate
+file paths for the HDD tier.
+* `tachyon.worker.tieredstore.level1.dirs.quota=2TB,5TB,500GB` defines the quota for each of the 3
+file paths of the HDD tier.
+
+There are a few restrictions to defining the tiers. First of all, there can be at most 3 tiers.
+Also, at most 1 tier can refer to a specific alias. For example, at most 1 tier can have the alias
+HDD. If you want Tachyon to use multiple hard drives for the HDD tier, you can configure that as
+multiple storage directories in the HDD tier.
+
+Additionally, the specific evictor and allocator strategies can be configured. Those configuration
+parameters are:
+
+    tachyon.worker.allocate.strategy.class=tachyon.worker.block.allocator.MaxFreeAllocator
+    tachyon.worker.evict.strategy.class=tachyon.worker.block.evictor.LRUEvictor
+
+# Configuration Parameters For Tiered Storage
+
+These are the configuration parameters for tiered storage.
+
+<table class="table-striped">
+<tr><th>Parameter</th><th>Default Value</th><th>Description</th></tr>
+<tr>
+  <td>tachyon.worker.tieredstore.level.max</td>
+  <td>1</td>
+  <td>
+  The maximum number of storage tiers in Tachyon. Currently, Tachyon supports 1, 2, or 3 tiers.
+  </td>
+</tr>
+<tr>
+  <td>tachyon.worker.tieredstore.level{x}.alias</td>
+  <td>MEM (for tachyon.worker.tieredstore.level0.alias)</td>
+  <td>
+  The alias of each storage tier, where x represents storage tier number (top tier is 0). Currently,
+  there are 3 aliases, MEM, SSD, and HDD.
+  </td>
+</tr>
+<tr>
+  <td>tachyon.worker.tieredstore.level{x}.dirs.path</td>
+  <td>/mnt/ramdisk (for tachyon.worker.tieredstore.level0.dirs.path)</td>
+  <td>
+  The paths of storage directories in storage tier x, delimited by comma. x represents the storage
+  tier number (top tier is 0). It is suggested to have one storage directory per hardware device for
+  the SSD and HDD tiers.
+  </td>
+</tr>
+<tr>
+  <td>tachyon.worker.tieredstore.level{x}.dirs.quota</td>
+  <td>128MB (for tachyon.worker.tieredstore.level0.dirs.quota)</td>
+  <td>
+  The quotas for all storage directories in storage tier x, delimited by comma. x represents the
+  storage tier number (starting from 0). For a particular storage tier, if the list of quotas is
+  shorter than the list of directories of that tier, then the quotas for the remaining directories
+  will just use the last-defined quota. Quota definitions use these suffixes: KB, MB, GB, TB, PB.
+  </td>
+</tr>
+<tr>
+  <td>tachyon.worker.allocate.strategy.class</td>
+  <td>tachyon.worker.block.allocator.MaxFreeAllocator</td>
+  <td>
+  The class name of the allocation strategy to use for new blocks in Tachyon. Currently, the
+  possible strategies are GREEDY and MAX_FREE.
```

Comment:
What about replacing `GREEDY` and `MAX_FREE` with `tachyon.worker.block.allocator.GreedyAllocator` and `tachyon.worker.block.allocator.MaxFreeAllocator`? Seems more direct for users to know other options of allocators.